### PR TITLE
Added hardware and serial for EatonUPS with management card PN:744-A3…

### PR DESCRIPTION
**Pull request by yvesboudreau**
…983-02

Please give a short description what your pull request is for

This add hardware model and serial number to the device for Eaton UPS with OID: .1.3.6.1.4.1.534.1

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/polling/os/eatonups.inc.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+//EATON-OIDS::xupsMIB.14.5.0 = STRING: "G311J11112"
+
+
+// This add the software version of the UPS and the management card separated by a "/"
+$version = trim(snmp_get($device, 'xupsIdentSoftwareVersion.0', '-OQv', 'XUPS-MIB'), '" ');
+$version .= ' / '.trim(snmp_get($device, 'xupsMIB.14.3.0', '-OQv', 'EATON-OIDS'), '" ');
+
+// This add the hardware of the UPS and the management card separated by a "/"
+$hardware  = trim(snmp_get($device, 'xupsIdentModel.0', '-OQv', 'XUPS-MIB'), '" ');
+$hardware  .= ' / '.trim(snmp_get($device, 'xupsMIB.14.4.0', '-OQv', 'EATON-OIDS'), '" ');
```
Comment (PipoCanaja) [inline includes/polling/os/eatonups.inc.php:24]: Hi,
You should check that the snmpget return a value before using it. Currently, when the OID is not available, $hardware ends up with " / " as a value. Same goes with $serial btw.

Comment (yvesboudreau) [reply includes/polling/os/eatonups.inc.php]: I added a few check before committing the changes. Would that work?

Comment (PipoCanaja) [reply includes/polling/os/eatonups.inc.php]: Could you please attach the test-data ? That will allow regression tests to take place. 
I'll have a look at your code asap.

Comment (yvesboudreau) [reply includes/polling/os/eatonups.inc.php]: I added the tests files snmprec and json.
Hope that works out

Comment (yvesboudreau) [reply includes/polling/os/eatonups.inc.php]: is anything missing to make this work?

**Comment by Jellyfrog** [comment]
This has a merge conflict, if you solve it, feel free to reopen!

**Pull request closed without merging**